metrics/heapster.go

```diff
@@ -121,14 +121,13 @@ func createAndRunAPIServer(opt *options.HeapsterRunOptions, metricSink *metricsi
 		glog.Errorf("Could not create the API server: %v", err)
 		return
 	}
-	healthz.InstallHandler(server.Mux, healthzChecker(metricSink))
```

Comment:
I thought when I was looking there was something in the `PrepareRun()` code of the generic API server code that did this for us